compiler/fir/analysis-tests/testData/resolve/arguments/operatorsOverLiterals.kt

```diff
@@ -20,7 +20,7 @@ fun test_2(n: Int) {
 
 fun Int.bar(): Int {}
 
-fun Int.baz(): Int
+fun Int.baz(): Int {}
```

Comment:
Otherwise it would be `NON_MEMBER_FUNCTION_NO_BODY`, but I don't think the purpose of this test is to check that.